kong/plugins/acme/schema.lua

```diff
@@ -34,27 +38,44 @@ local SHM_STORAGE_SCHEMA = {
 local KONG_STORAGE_SCHEMA = {
 }
 
-local REDIS_STORAGE_SCHEMA = {
-  { host = typedefs.host, },
-  { port = typedefs.port, },
-  { database = { type = "number", description = "The index of the Redis database to use.", } },
```

Comment:
it looks like that these options went away, right? However, I need to be able to configure the plugin using the old options. Yet, it should show me a deprecation warning

Reply:
They didn't actually go away. They are available just the same but right now they come from `kong.tools.redis.schema`. The ones that differ from the common configuration are defined in `REDIS_LEGACY_SCHEMA_FIELDS`